drivers/mlx90393/include/mlx90393_constants.h

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright (C) 2023 mi6527ri
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers_mlx90393
+ * @{
+ *
+ * @file
+ * @brief       MLX90393 internal addresses, registers and constants
+ *
+ * @author      Michael Ristau <michael.ristau@fh-erfurt.de>
+ */
+
+#ifndef MLX90393_CONSTANTS_H
+#define MLX90393_CONSTANTS_H
+
+#ifdef __cplusplus
+extern "C" //{
+#endif
+
+/**
+ * @brief   Register address map
+ */
+#define MLX90393_REG_CONF0          0x00    /**< Configuration register 0 for several parameters **/
```

Comment:
```suggestion
#define MLX90393_REG_CONF0          0x00    /**< Configuration register 0 */
```
Bei allen Kommentaren in dieser Datei, bitte `**/` global durch `*/` ersetzen und die Einhaltung der maximalen Zeichenzahl 100 einhalten.